Python/pywarpx/_libwarpx.py

```diff
@@ -74,8 +74,13 @@ def _get_package_root():
 
 try:
     libwarpx = ctypes.CDLL(os.path.join(_get_package_root(), libname))
-except OSError:
-    raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname)
+except OSError as e:
+    value = e.args[0]
+    if f'{libname}: cannot open shared object file: No such file or directory' in value:
+        raise Exception('"%s" was not installed. It can be installed by running "make" in the Python directory of WarpX' % libname) from e
+    else:
+        print("Failed to load the libwarpx shared object library")
```

Comment:
Can we raise the error text in `e` as well here to potentially add more context?

Does this work?
```suggestion
        print("Failed to load the libwarpx shared object library: {0}".format(e))
```

Reply:
The line `OSError: <library name>: cannot open shared object file: No such file or directory` is printed out at the end of the stack trace. 

Printing out the exception message again seems a little redundant, so maybe just printing out the name of the shared library would be better: `"Failed to load the libwarpx shared object library: <library name>"` instead of printing:

`"Failed to load the libwarpx shared object library: <library name>: cannot open shared object file: No such file or directory"`

Because that line already appears after the stack trace.

Reply:
Oh, the `raise` will print that. Got it :)